addons/hearing/initKeybinds.inc.sqf

```diff
@@ -0,0 +1,17 @@
+["ACE3 Equipment", QGVAR(putOrRemoveEarplugs), localize LSTRING(PutOrRemoveEarplugs), {
+    // Conditions: specific
+    if !([ACE_player, objNull, ["isNotSwimming", "isNotInside", "isNotSitting"]] call EFUNC(common,canInteractWith)) exitWith {false};
+
+    if (GVAR(EnableCombatDeafness) && {!([ACE_player] call FUNC(hasEarPlugsIn)) && {'ACE_EarPlugs' in items ACE_player}}) exitWith {
+        [ACE_player,true] call FUNC(putInEarPlugs);
+        true;
+    };
+    if (GVAR(EnableCombatDeafness) && {[ACE_player] call FUNC(hasEarPlugsIn)}) exitWith {
+        [ACE_player,true] call FUNC(removeEarPlugs);
+        true;
+    };
+    
+    false
+},
+{false},
+[35, [false, false, true]], true] call CBA_fnc_addKeybind; //(ALT + H)
```

Comment:
Removed default keybind, added new line at the end of the file and changed some code formatting, as well as some functions:
```suggestion
["ACE3 Equipment", QGVAR(putOrRemoveEarplugs), LLSTRING(PutOrRemoveEarplugs), {
    // Conditions: specific
    if !([ACE_player, objNull, ["isNotSwimming", "isNotInside", "isNotSitting"]] call EFUNC(common,canInteractWith)) exitWith {false};

    if (GVAR(EnableCombatDeafness) && {!([ACE_player] call FUNC(hasEarPlugsIn))} && {[ACE_player, "ACE_EarPlugs"] call EFUNC(common,hasItem)}) exitWith {
        [ACE_player, true] call FUNC(putInEarPlugs);
        true
    };
    if (GVAR(EnableCombatDeafness) && {[ACE_player] call FUNC(hasEarPlugsIn)}) exitWith {
        [ACE_player, true] call FUNC(removeEarPlugs);
        true
    };
    
    false
}] call CBA_fnc_addKeybind; // UNBOUND

```